scRNA-seq/02-tag-based_pre-processing_scRNA-seq.md

```diff
@@ -4,12 +4,14 @@
 
 #### In this section, we will be running through the basics of pre-processing single-cell RNA-seq data.
 
-We will be using a peripheral blood mononuclear cells (pbmc's) 10X Genomics [(Zheng et al, 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601) dataset as an example.
-For 10X Genomics scRNA-seq data, cells are separated by emulsion/droplets, and
-individual cells are given barcodes.
+We will be using a tag-based scRNA-seq sample from [Tabula Muris data](https://www.nature.com/articles/s41586-018-0590-4).
+This dataset is made of 20 mouse organs that were sequenced using 10X Genomics 
+Chromium single cell sequencing methods. 
+In these methods, cells are separated by emulsion/droplets, and individual cells
```

Comment:
I think `In these methods` is less clear than what you had earlier `For 10X Genomics scRNA-seq data`